public_html/estacada.html

```diff
@@ -1321,6 +1345,9 @@ <h2 class="inline-block">Location</h2>
                                 Located in Mt. Hood Territory, Estacada has the Mt. Hood National Forest and Clackamas River recreation area as its backyard, and is only a 45 minute drive to downtown Portland. The city is know for it's thriving arts culture, friendly downtown shops, local farmers market, and abundant outdoor recreation opportunities. The Estacada High School was recently ranked a top high school by US News and World Report. For more information visit the Chamber of Commerce's <a href="http://www.estacadachamber.org/main/" target="_blank">website</a>.
                             </p>
                             <h2 class="inline-block">Notice of Privacy Practices</h2>
+                            <p>
+                                <a href="learn\mckenzie\Orchid_Health_Notice_of_Privacy_Practices.pdf" target="_blank">Notice of Privacy Practices</a>
```

Comment:
since this document is more general, can we move it to just the learn/ folder instead of learn/mckenzie/